variant-remapping-automation/genome_target_tracker.py

```diff
@@ -0,0 +1,96 @@
+from argparse import ArgumentParser
+
+import requests
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_ensembl
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+remapping_genome_target_table = 'evapro.remapping_genome_target_tracker'
+ensembl_url = 'http://rest.ensembl.org/info/assembly'
+
+
+def get_all_taxonomies_from_eva(private_config_xml_file):
+    taxonomy_list = []
+    with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
+        query = 'SELECT DISTINCT taxonomy_id FROM evapro.taxonomy'
+        for taxonomy in get_all_results_for_query(pg_conn, query):
+            taxonomy_list.append(taxonomy[0])
+
+    return taxonomy_list
+
+
+def get_tax_asm_from_eva(private_config_xml_file):
+    eva_tax_asm = {}
+    with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
+        query = f"""SELECT DISTINCT taxonomy_id, assembly_id FROM {remapping_genome_target_table} 
+        WHERE current_previous='current'"""
+        for tax_id, asm in get_all_results_for_query(pg_conn, query):
+            eva_tax_asm[tax_id] = asm
+
+    return eva_tax_asm
+
+
+def get_tax_asm_from_ensembl(taxonomy_list):
+    ensembl_tax_asm = {}
+    for tax_id in taxonomy_list:
+        try:
+            logger.info(f'Query Ensembl for species name using taxonomy {tax_id}')
+            sp_name = get_scientific_name_from_ensembl(tax_id)
+        except Exception:
+            logger.warning(f'Could not get species name for taxonomy {tax_id}')
+            continue
+        logger.info(f'Query ensembl for supported assembly for taxonomy {tax_id}')
+        asm = get_supported_asm_from_ensembl(sp_name)
+        if asm != 'None':
+            ensembl_tax_asm[tax_id] = asm
+
+    return ensembl_tax_asm
+
+
+def get_supported_asm_from_ensembl(scientific_name):
+    url = ensembl_url + '/' + scientific_name.lower().replace(' ', '_')
+    response = requests.get(url, params={'content-type': 'application/json'})
+    data = response.json()
+    assembly_accession = str(data.get('assembly_accession'))
+    return assembly_accession
+
+
+def check_supported_target_assembly(private_config_xml_file):
+    taxonomy_list = get_all_taxonomies_from_eva(private_config_xml_file)
+    eva_tax_asm = get_tax_asm_from_eva(private_config_xml_file)
+    ensembl_tax_asm = get_tax_asm_from_ensembl(taxonomy_list)
+
+    taxonomy_not_present_in_eva = []
```

Comment:
This name seems misleading to me. I think you mean taxonomy from EVA that are not yet tracked in the `remapping_genome_target_tracker`  don't you ? 
If yes I would suggest something like `taxonomy_not_tracked_by_remapping`

Reply:
renamed